### PR TITLE
Make Firebase calls to hackathons more dynamic

### PR DESCRIPTION
**Pull request by tdanielles**
## Description
Ticket: [https://www.notion.so/nwplus/PA-04-Make-firebase-calls-dynamic-09c5c5e88a85410e82f402d9efa63d8c?pvs=4](https://www.notion.so/nwplus/PA-04-Make-firebase-calls-dynamic-09c5c5e88a85410e82f402d9efa63d8c?pvs=4)

- Made firebase calls more dynamic by using the `dbHackathonName` value instead of the hardcoded constant `DB_HACKATHON` 😄

### CODE REVIEW

File: src/containers/Faq.jsx [hunk 1]
```diff
@@ -15,16 +16,17 @@ const FAQ = () => {
...
     const unsubscribe = db
       .collection(FAQ_COLLECTION)
-      .where('hackathonIDs', 'array-contains-any', ['Portal'])
+      .where('hackathonIDs', 'array-contains-any', [dbHackathonName])
       .onSnapshot(querySnapshot => {
         setFaqs(Object.values(querySnapshot.docs.map(doc => doc.data())))
       })
     return unsubscribe
-  }, [setFaqs])
+  }, [setFaqs, dbHackathonName])
```
Comment (martincai8) [inline src/containers/Faq.jsx:29]: can probably remove `setFaqs` in the array here, i don't think it does anything

File: src/utility/firebase.js [hunk 2]
```diff
@@ -10,6 +10,9 @@ import {
   HACKER_APPLICATION_TEMPLATE,
   REDIRECT_STATUS,
 } from '../utility/Constants'
+import { useHackathon } from './HackathonProvider'
+
+// const { activeHackathon } = useHackathon()
```
Comment (martincai8) [inline src/utility/firebase.js:15]: can probably remove these as well!

**Review by martincai8 — APPROVED** [review]
lgtm overall! i think there are still some other firebase calls that rely on `DB_HACKATHON` (e.g., `currentHackathonRef`, `applicantsRef`, `projectsRef`, `announcementsRef` in `firebase.js`), but i can fix those if you want so you can focus on the cms stuff 🚀

**Review by joannelex — APPROVED** [review]

**Review by martincai8 — APPROVED** [review]
lgtm again 💪 i think we're good to remove `export const DB_HACKATHON = 'cmd-f2024'` from `Constants.js` now

**Pull request merged**